bitnami/kubeapps/templates/apprepository/rbac.yaml

```diff
@@ -98,7 +98,7 @@ rules:
 apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
 kind: Role
 metadata:
-  name: {{ .Release.Name }}-repositories-write
+  name: {{ printf "%s-repositories-write" .Release.Name }}
```

Comment:
Other metadata names are using the `quote` pipe. Is there any reason why this one is not?
eg:

```
name: {{ printf "kubeapps:%s:apprepositories-read" .Release.Namespace | quote }}
````